src/stormpath.social-login.facebook.js

```diff
@@ -0,0 +1,67 @@
+(function() {
+  'use strict';
```

Comment:
Strict is only applied to the scoped part, and not the whole file. Not better to have it first?